### PR TITLE
[298] - Record if they have shared disability details or not

### PR DESCRIPTION
**Pull request by darokel**
### Context

- https://trello.com/c/nhV0FCuw/298-s-record-if-they-have-shared-disability-details-or-not

### Changes proposed in this pull request

- Implement the disability disclosure section of the diversity workflow
- Piece together the routes connecting these sections. Ie. if no ethnic group is selected go to disability disclosure and make sure if you click back you return to the ethnic group selection

Screenshot 2020-10-29 at 13 10 19

Screenshot 2020-10-29 at 13 10 31

### Guidance to review

- Visit the review app
- Begin filling out the diversity section
- Assert not providing an ethnic group takes you to the disability disclosure.
- Assert choosing an ethnic group and background takes you to the disability disclosure

### CODE REVIEW

**Comment by darokel** [comment]
[[S] Record if they have shared disability details or not](https://trello.com/c/nhV0FCuw/298-s-record-if-they-have-shared-disability-details-or-not)

File: app/views/trainees/diversity/disability_disclosures/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: "Back",
+    href: request.referrer
```
Comment (defong) [inline app/views/trainees/diversity/disability_disclosures/edit.html.erb:4]: We need to be explicit with the path and not infer form the client.

It broken now

https://dfe-twd-rttd-pr-124.herokuapp.com/trainees/3/diversity/disability-disclosure/edit

**Review by defong — CHANGES_REQUESTED** [review]

File: app/views/trainees/diversity/disability_disclosures/edit.html.erb (hunk 1 above)
Comment (darokel) [reply app/views/trainees/diversity/disability_disclosures/edit.html.erb:4]: Forgot to clean this up thanks for spotting. I set that for testing 🙈

Comment (defong) [reply app/views/trainees/diversity/disability_disclosures/edit.html.erb]: fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/124/commits/c538876731f3309572ed3562fb85e67cf0347b98

**Review by dankmitchell — APPROVED** [review]

**Review by aliuk2012 — APPROVED** [review]

**Pull request merged**